### PR TITLE
feat(alerts): weather-contextualized ford crossing alert

### PR DESCRIPTION
**Pull request by vincentchalamon**
## Contexte

PR2 (finale) de l'alerte **gué**, contextualisée météo (consomme `osm.fords` importé en #699).

## Changements

**API**
- **`FordRepository`** : gués `osm.fords` à ≤ **25 m** du tracé d'étape (`ST_DWithin`).
- **`CheckFordsHandler`** : par étape, **nudge** « gué sur le parcours » par temps sec, **warning** « gué + pluie prévue : prudence » si `precipitationProbability >= 50%`. Dédup par nom, action navigate, via `MercureEventType::FORD_ALERTS`.
- **`ComputationName::FORDS`** (catégorie `terrain_security`) **dispatché après la météo** par `FetchWeatherHandler` (comme `AnalyzeWind`), pour lire `stage.weather` de façon fiable — pas dans `TripAnalysisDispatcher`.

**Frontend** : cas SSE `ford_alerts` (`nudge`|`warning` + action) + typage.

**i18n** : `alert.ford.{nudge,warning,action}` (en + fr). **Doc** : 2 lignes table alertes README + `ALERT_RULE_MAP` + fixture `trip-schema.json` (computation `fords`).

## Tests

- `CheckFordsHandlerTest` : nudge sec, warning pluie, météo absente→nudge, jour de repos sauté, stages null.
- `FordIndexReadTest` (intégration PostGIS) : détection à proximité, rien sinon, étape dégénérée.
- `AlertDocumentationTest` : namespace `ford` mappé + README.
- **Local** : PHPStan L9, Rector, CS-Fixer clean ; tsc + ESLint front clean ; unit verts. Tests DB via CI.

## Clôture

Termine les alertes d'enrichissement 2e vague (col abandonné, ferry #697/#698, gué #699/celle-ci). Reste, hors alertes : **#15 Wikidata au provisioner**.

## Claude Review

**Summary:** Clean, well-structured addition that faithfully follows the ferry/border-crossing handler pattern. The dispatch-after-weather architecture is correctly motivated. All previous findings have been addressed — the boundary-value test was added in the second commit.

**Resolved threads:** Resolved 1 previously open thread (`precipitationProbability == 50` boundary — now covered by `escalatesToAWarningAtExactThreshold`).

**PR title check:** `feat(alerts): weather-contextualized ford crossing alert` — description should use imperative mood per project convention: `add weather-contextualized ford crossing alert`.

**Review checklist:**
- [x] Code respects the project architecture
- [x] SOLID principles and Law of Demeter followed
- [x] Design patterns used where appropriate
- [x] Tests cover new/changed cases
- [x] Documentation is up to date
- [x] Dependent tickets accounted for (#699)

**Inline comments:** No inline comments.

Reviewed commit: `59244fb5cdd9b6e84d5a843b0b9af897d29a3e82`

### CODE REVIEW

File: api/tests/Unit/MessageHandler/CheckFordsHandlerTest.php [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        $handler(new CheckFords('trip-1'));
+    }
+
+    #[Test]
+    public function treatsMissingForecastAsDry(): void
+    {
+        // No weather on the stage → no rain info → nudge, not warning.
+        $tripStateManager = $this->createTripStateManager([$this->stage(1)]);
+
+        $publisher = $this->createMock(TripUpdatePublisherInterface::class);
```
[the remark this answers is not in the document]
Comment (vincentchalamon) [reply api/tests/Unit/MessageHandler/CheckFordsHandlerTest.php:176]: Ajouté (59244fb5) : `escalatesToAWarningAtExactThreshold` couvre `precipitationProbability == 50` (le cut-point `>= 50`) → warning. Une régression sur la constante ou l'opérateur serait捕rapée.
